src/main/java/org/codehaus/groovy/control/CompilerConfiguration.java

```diff
@@ -66,6 +66,9 @@ public class CompilerConfiguration {
     /** Joint Compilation Option for enabling generating stubs in memory. */
     public static final String MEM_STUB = "memStub";
 
+    /** Optimization Option for enabling dead code analysis */
+    public static final String ANALYZE_DEAD_CODE = "analyzeDeadCode";
+
```

Comment:
Can you move this adjacent to the other optimization options and add an `@since` tag?

I'm partial to naming it `DEAD_CODE_ANALYSIS` and `"deadCodeAnalysis"`.